### PR TITLE
VS 2013

### PR DESCRIPTION
**Pull request by codewerfer**
fixes for VS 2013

### CODE REVIEW

File: src/gsCore/gsBoxTopology.cpp [hunk 1]
```diff
@@ -283,12 +283,12 @@ std::vector< std::vector<patchComponent> > gsBoxTopology::allComponents(bool com
     for (short_t i=dim; i>=0; --i)
     {
         if (!combineCorners || i>0)
-            for( typename map_t::iterator it = comps[i].begin(); it != comps[i].end(); ++it )
+            for( /*typename*/ map_t::iterator it = comps[i].begin(); it != comps[i].end(); ++it )
```
Comment (stefantakacs) [inline src/gsCore/gsBoxTopology.cpp]: why do you not remove typename here?

File: src/gsModeling/gsParametrization.hpp [hunk 2]
```diff
@@ -537,7 +537,7 @@ gsParametrization<T>::LocalParametrization::LocalParametrization(const gsHalfEdg
                 length = (*meshInfo.getVertex(indices.front()) - *meshInfo.getVertex(m_vertexIndex)).norm();
                 //length =  (meshInfo.getVertex(indices.front()) - meshInfo.getVertex(m_vertexIndex) ).norm();
                 nextAngle = angles.front()*thetaInv * 2 * EIGEN_PI;
-                nextVector = ((Eigen::Rotation2D<T>(nextAngle) * actualVector).normalized()*length) + p;
+                nextVector = ((Eigen::Rotation2D<T>(nextAngle).operator*(actualVector)).normalized()*length) + p;
```
Comment (stefantakacs) [inline src/gsModeling/gsParametrization.hpp]: What's the underlying bug here?

File: src/gsCore/gsBoxTopology.cpp (hunk 1 above)
Comment (filiatra) [reply src/gsCore/gsBoxTopology.cpp]: yes please take it out

File: src/gsModeling/gsParametrization.hpp (hunk 2 above)
Comment (codewerfer) [reply src/gsModeling/gsParametrization.hpp]: fixes this: https://cdash.ricam.oeaw.ac.at/viewBuildError.php?buildid=348488

Comment (stefantakacs) [reply src/gsModeling/gsParametrization.hpp]: thanks. i still do not get it why this helps, but fine...

File: src/gsCore/gsBoxTopology.cpp (hunk 1 above)
Comment (stefantakacs) [reply src/gsCore/gsBoxTopology.cpp]: done.

**Comment by codewerfer** [comment]
Changes are already in https://github.com/gismo/gismo/pull/328

**Pull request closed without merging**